plan/typeinferer_test.go

```diff
@@ -122,6 +122,7 @@ func (ts *testTypeInferrerSuite) TestInferType(c *C) {
 		// Functions
 		{"version()", mysql.TypeVarString, charset.CharsetUTF8, 0},
 		{"count(c_int)", mysql.TypeLonglong, charset.CharsetBin, mysql.BinaryFlag},
+		{"abs()", mysql.TypeNull, charset.CharsetBin, mysql.BinaryFlag},
```

Comment:
I think since you've modified the type inferer of these functions,
it better to test it in the same PR.